app/test_split_integer.py

```diff
@@ -2,20 +2,24 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(20, 3)) == 20
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(21, 3)
+    assert (max(result) - min(result)) == 0
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result = split_integer(20, 1)
+    assert result == [20]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    result = split_integer(20, 3)
+    assert split_integer(20, 1) == [20]
```

Comment:
The assertion `split_integer(20, 1) == [20]` is incorrect in this context. It seems like a copy-paste error from the previous test. You should assert that the result is sorted, for example, `assert result == sorted(result)`.